script/e2e-test.sh

```diff
@@ -260,16 +281,12 @@ img_flags=(
   "--set" "kubeappsapis.image.repository=${images[5]}"
 )
 
+additional_flags_file=$(generateAdditionalValuesFile)
+
 if [ "$USE_MULTICLUSTER_OIDC_ENV" = true ]; then
   multiclusterFlags=(
     "--set" "ingress.enabled=true"
-    "--set" "ingress.hostname=localhost"
-    "--set" "ingress.tls=true"
-    "--set" "ingress.selfSigned=true"
-    "--set" "ingress.annotations.\"kubernetes\.io\/ingress\.class\"=nginx"
-    "--set" "ingress.annotations.\"nginx\.ingress\.kubernetes\.io\/proxy-buffer-size\"=8k"
-    "--set" "ingress.annotations.\"nginx\.ingress\.kubernetes\.io\/proxy-read-timeout\"=600.0"
-    "--set" "ingress.annotations.\"nginx\.ingress\.kubernetes\.io\/proxy-buffers\"=4.0"
```

Comment:
I'd rather move all the `ingress.xxx` flags to the file OR just the `ingress.annotations.xxx`, but leaving the enabled=true as a `--set` and the rest on the file... seems a bit odd, no? 

I'm for moving just the annotations, but see what you think.

Thanks for the fix, btw!

Reply:
Ok, initially I left it there on purpose to clearly show that the ingress was enabled. Let's have everything related to ingress in that additional file, it is strange to have ingress settings scattered. WDYT?